Creation of the zh-CN.json translation file

tenbrinck opened the pull request:
In this commit, the translation file zh-CN.json was added to provide support for Chinese (Mandarin). This extends the multilingual capability of the application and allows users to use the application in their native language.

tenbrinck committed:
Creation of the zh-CN.json translation file

In this commit, the translation file zh-CN.json was added to provide support for Chinese (Mandarin). This extends the multilingual capability of the application and allows users to use the application in their native language.

fb-sean commented on zh-CN.json:
```diff
@@ -0,0 +1,284 @@
+  },
+  "verification": {
+    "infoMobile": "移动用户信息",
+    "infoMobileText": "如果你是移动用户。请在你的浏览器上打开https://kick.com并在浏览器中进行步骤。Kick应用目前不支持资料更改。",
```
space needed between the link else Discord thinks its 1 link

fb-sean replied:
^ counts for all links

fb-sean commented on zh-CN.json:
```diff
@@ -0,0 +1,284 @@
+      "title": "订阅者角色",
+      "description": "要获得**{streamer}**的订阅者角色，你只需要点击下面名为**Sub Role**的按钮。"
+    },
+    "added": "我已经将订阅者角色嵌入发送到了<#{channel}>！\n*信息：如果你想要删除设置，只需删除消息和角色，机器人将自动从数据库中删除设置。*",
```
just noticed also for every placeholder or like <#id> too this all needs spaces else Discord does not render it

tenbrinck committed:
Add Russian translation with adjusted link spacing

This commit introduces the Russian translation for the configuration file, ensuring that spaces are properly inserted between links to address the issue where Discord interprets them as a single link. This adjustment improves user interaction with the application within Discord by enabling correct recognition and navigation of provided links. The translation covers all necessary UI elements and system messages,

tenbrinck committed:
Update zh-CN.json

tenbrinck committed:
Update zh-CN.json

tenbrinck committed:
Delete zh-CN.json

tenbrinck committed:
Create zh-CN.json
